cognite_toolkit/_cdf_tk/client/data_classes/agents.py

```diff
@@ -0,0 +1,203 @@
+
+    Args:
+        id (int): A server-generated ID for the object.
+        external_id (str): The external ID provided by the client. Must be unique for the resource type.
+        name (str): The name of the agent.
+        description (str): The description of the agent.
+        owner_id (str): The owner ID of the agent.
+        instructions (str): Instructions for the agent.
+        model (str | None): Name of the language model to use.
+        labels (list[str] | None): List of labels for the agent.
+        example_questions (list[dict[str, str]]) | None: List of example questions for the agent.
+        tools (list[AgentTool] | None): List of tools for the agent.
+    """
+
+    def __init__(
+        self,
+        id: int,
+        external_id: str,
+        name: str,
+        description: str,
+        owner_id: str,
+        instructions: str,
+        model: str,
+        labels: list[str] | None = None,
+        example_questions: list[dict[str, str]] | None = None,
+        tools: list[AgentTool] | None = None,
+    ) -> None:
+        self.id = id
+        self.external_id = external_id
+        self.name = name
+        self.description = description
+        self.owner_id = owner_id
+        self.instructions = instructions
+        self.model = model
+        self.labels = labels
+        self.example_questions = example_questions
+        self.tools = tools
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        result = super().dump(camel_case=camel_case)
+        if self.tools:
+            result["tools"] = [item.dump(camel_case=camel_case) for item in self.tools]
+        return result
+
+
+class Agent(AgentCore):
+    """Representation of an AI Agent in CDF.
+    This is the read format of an agent.
+
+    Args:
+        id (int): A server-generated ID for the object.
+        external_id (str): The external ID provided by the client. Must be unique for the resource type.
+        name (str): The name of the agent.
+        description (str): The description of the agent.
+        owner_id (str): The owner ID of the agent.
+        instructions (str): Instructions for the agent.
+        model (str | None): Name of the language model to use.
+        labels (list[str] | None): List of labels for the agent.
+        example_questions (list[dict[str, str]]) | None: List of example questions for the agent.
+        tools (list[AgentTool] | None): List of tools for the agent.
+    """
+
+    def __init__(
+        self,
+        id: int,
+        external_id: str,
+        name: str,
+        description: str,
+        owner_id: str,
+        instructions: str,
+        model: str,
+        labels: list[str] | None = None,
+        example_questions: list[dict[str, str]] | None = None,
+        tools: list[AgentTool] | None = None,
+    ) -> None:
+        super().__init__(
+            id, external_id, name, description, owner_id, instructions, model, labels, example_questions, tools
+        )
```

Comment:
Are there no server set properties? If not this entire method can be skipped as it is implemented in the parent
```suggestion
```